TestCases/transition/E387_Airfoil/transitional_SST_LM_model_ConfigFile.cfg

```diff
@@ -0,0 +1,150 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%                                                                              %
+% SU2 configuration file                                                       %
+% Case description: Eppler E387 Airfoil at Re = 200K                           %
+% Author: Andrea Rausa	                                                       %
+% Institution: Politecnico di Milano                                           %
+% Date: Dec 07th, 2022                                                         %
+% File Version 7.5.0 "Blackbird"                                               %
```

Comment:
Are you adding the airfoil cases to parallel_regression.py?

Reply:
Not yet, but I'll do it right now.

Reply:
The regression test case fails on GitHub but works fine on my PC. I also uploaded the same case in the serial_regression.py. How can I check which values GitHub computes?